envs/doom/multiplayer/doom_multiagent.py

```diff
@@ -97,9 +97,13 @@ def _ensure_initialized(self):
                 '+sv_nojump 1',  # Disables jumping.
                 '+sv_nofreelook 1',  # Disables free look with a mouse (only keyboard).
                 '+sv_noexit 1',  # Prevents players from exiting the level in deathmatch before timelimit is hit.
-                f'+viz_respawn_delay {self.respawn_delay}',  # Sets delay between respanws (in seconds).
-                '+viz_connect_timeout 4',  # In seconds
+                f'+viz_respawn_delay {self.respawn_delay}'  # Sets delay between respanws (in seconds).
```

Comment:
It's good to have a comma for the last element in the list, and Python allows this.
Minor stylistic thing since I'm requesting another change anyway